cmd/livegrep-fetch-reindex/main.go

```diff
@@ -26,8 +28,17 @@ var (
 	flagReloadBackend = flag.String("reload-backend", "", "Backend to send a Reload RPC to")
 	flagNumWorkers    = flag.Int("num-workers", 8, "Number of workers used to update repositories")
 	flagNoIndex       = flag.Bool("no-index", false, "Skip indexing after fetching")
+	flagMetricsPath   = flag.String("metrics-out", "", "File that indexing metrics should be sent to")
```

Comment:
Looks like these were left and probably not intended to be part of this PR?

Reply:
ah, good catch, removed.